### DISCUSSION TITLE
Reverse proxy MUX parameters

### DISCUSSION THREAD

**Discussion by pulsarice** [XTLS/Xray-core#3321] [Q&A]
Hi
I receive this error on my portal server from time to time
`[Error] common/mux: failed to read metadata > common/mux: invalid metalen 5891`
and this on my bridge server
`[Error] common/mux: unexpected EOF > common/mux: failed to read metadata > common/mux: invalid metalen 5891`
Bridge is connected to portal servers via Cloudflare CDN with vless+ws+tls.
Per Xray-core documentation MUX is enabled by default on reverse proxies and I didn't enable it on my bridge server outbound.

Are there any user adjustable MUX parameters for reverse connections?

**Comment by sega-gremlen** [comment]
i have the same, did you get why it happens??
